charts/pomerium/templates/authorize-deployment.yaml

```diff
@@ -68,6 +68,11 @@ spec:
         env:
         - name: SERVICES
           value: authorize
+        - name: SIGNING_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.proxy.signingKeySecret.name" . }}
```

Comment:
This parameter was actually moved from `proxy` to `authorize` service in the v0.7 release.  

Would you mind taking a pass at relocating the related structures under `authorize` and removing it from the proxy service settings?

We could consider this a minor version change.